src/data/title-requirements.csv

```diff
@@ -1,138 +1,138 @@
-"どちらかが育成で無敗で皐月賞、日本ダービー、菊花賞を\n勝利したウマ娘から想いを継承し、\n無敗で皐月賞、ダービー、菊花賞を勝利する","Inherit from parents with no record of defeat in Satsuki Sho, Japan Derby, Kikka Sho and win said races while being undefeated"
-"どちらも育成ランク[C]以下のウマ娘から想いを継承し、\n育成ランクA以上で育成を完了する",""
-"どちらかが育成で桜花賞、オークス、秋華賞を\n勝利したウマ娘から想いを継承し、\n桜花賞、オークス、秋華賞を勝利する",""
-"どちらかが育成で無敗で桜花賞、オークス、秋華賞を\n勝利したウマ娘から想いを継承し、\n無敗で桜花賞、オークス、秋華賞を勝利する",""
-"どちらもファン数が460000人以上のウマ娘から想いを継承し、\nファン数が480000人以上になる",""
-"全レースで2着以内で、\n菊花賞、天皇賞（春）、宝塚記念を勝利し、\n目標を全て達成し育成を完了する",""
+"天気[雨]で4回以上出走し、3勝する","Participate a race with rainy weather 4 times or more\nWin at least 3 of them"
+"天気[曇り]で4回以上出走し、3勝する","Participate a race with cloudy weather 4 times or more\nWin at least 3 of them"
+"天気[雪]で3回以上出走し、2勝する","Participate a race with snowy weather 3 times or more\nWin at least 2 of them"
+"GⅠに1回以上出走し、\nGⅠ未勝利のまま育成ランクC以上で育成を完了する","<size=22>Participate a GI race once or more without winning any single GI race\nComplete the training with C or higher rank</size>"
+"ファン数が550000人以上になる","Earn a total of 550000 or more fans"
+"10回以上レースに出走し、\n芝のレースのみに出走して育成を完了する","<size=22>Participate a race 10 times or more\nEvery race has to be turf\nComplete the training</size>"
+"掛かったレースで3勝する","While under effect of temptation debuff\nWin a race 3 times"
+"どちらも育成で重賞未勝利のウマ娘から想いを継承し、\nGⅠを勝利する","Inherit memories from parents without any winning history\nWin a GI race"
+"作戦「逃げ」または「先行」で\nGⅡもしくはGⅠを3バ身差以上で合計6勝する","Win a GI or GII race 6 times as a runner or leader with difference of 3 or more horse lengths"
+"京都金杯、中山金杯、金鯱賞のいずれかで2勝する","The Kyoto Kinpai, Nakayama Kinpai, and Kinko Sho\nWin at least 2 of them"
+"トレーニング[パワー]がLv5になる","Reach power training level 5"
+"トレーニング[スタミナ]を35回行う","Reach stamina training level 5"
+"トレーニング[根性]がLv5になる","Reach guts training level 5"
+"トレーニング[賢さ]がLv4になる","Reach intelligence training level 5"
+"5連敗以上した後に勝利する","Win a race after 5 losses in a row"
+"メイクデビューを7バ身差以上で勝利する","Win the Make Debut with difference of 7 or more horse lengths"
+"トレーニングを一切しないでメイクデビューに勝利する","Win the Make Debut without any single training"
+"10回以上レースに出走し、\nトレーニングに1度も失敗しないで育成を完了する","<size=22>Participate a race 10 times without failing single training\nComplete the training</size>"
+"作戦「逃げ」で6勝し、\n基礎能力[スピード]が600以上になる","Win a race 6 times as a runner\nReach 600 or higher speed"
+"作戦「差し」または「追込」でGⅠを勝利し、\n基礎能力[パワー]と[根性]がそれぞれ500以上になる","Win a GI race once as a betweener or chaser\nReach 500 or higher power and guts"
+"中央10か所のレース場で、それぞれ重賞を勝利する","Win a big prize each in 10 central race grounds"
+"日本ダービーを勝利する","Win the Japan Derby"
+"オークスを勝利する","Win the Oaks"
+"皐月賞、日本ダービー、菊花賞のいずれかで2勝する","The Satsuki Sho, Japan Derby and Kikka Sho\nWin at least 2 of them"
+"桜花賞、オークス、秋華賞のいずれかで2勝する","The Oka Sho, Oaks and Shuka Sho\nWin at least 2 of them"
+"大阪杯、天皇賞（春）、宝塚記念を勝利する","Win the Oosaka Hai, Tenno Sho (Spring), and Takarazuka Kinen"
+"天皇賞（秋）、ジャパンC、有馬記念を勝利する","Win the Tenno Sho (Autumn), Japan Cup, and Arima Kinen"
+"天皇賞（春）と天皇賞（秋）を勝利する","Win the Tenno Sho (Spring) and Tenno Sho (Autumn)"
```

Comment:
I'm not sure about these; title requirements field have rather strict length limits. I'd check how they look like ingame and perhaps add size markup.

Reply:
most of them look fine, i put size markups for some longer requirements
https://files.catbox.moe/lukckg.PNG
https://files.catbox.moe/lrcaib.PNG